src/win/languages/en-US.rc

```diff
@@ -284,6 +284,7 @@ END
 #define STR_GLOBAL		"Save these settings as &global defaults"
 #define STR_DEFAULT		"&Default"
 #define STR_LANGUAGE		"Language:"
+#define STR_ICONSET "Iconset:"
```

Comment:
Indentation; also the dialog name should be changed if it's not solely display language selection anymore.

I wouldn't translate to other languages yet either and wait for the translations to be sure we don't ship some machine translated weirdness in the build. (This reminds me there is no fallback for missing strings in translations.)